src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/UnaryExpression.cs

```diff
@@ -38,14 +38,14 @@ internal UnaryExpression(ExpressionType nodeType, Expression expression, Type ty
         /// <summary>
         /// Gets the operand of the unary operation.
         /// </summary>
-        /// <returns> An <see cref="ExpressionType"/> that represents the operand of the unary operation.</returns>
+        /// <returns> An <see cref="Expression"/> that represents the operand of the unary operation. Returns null if node type is <see cref="ExpressionType.Throw"/> </returns>
```

Comment:
>  [](start = 165, length = 1)

Perhaps "... with no operand." to clarify that this only applies to re-throw.

Reply:
Sorry for not being clear. I meant: "Returns null if node type is ExpressionType.Throw with no operand." 

---
In reply to: [362924461](https://github.com/dotnet/runtime/pull/1064#discussion_r362924461) [](ancestors = 362924461)